modelmatrix-cli/src/main/scala/com/collective/modelmatrix/cli/db/InstallSchema.scala

```diff
@@ -1,29 +1,17 @@
 package com.collective.modelmatrix.cli.db
 
 import com.collective.modelmatrix.cli.Script
-import com.typesafe.config.ConfigFactory
-import org.flywaydb.core.Flyway
+import com.collective.modelmatrix.db.{DefaultDBConfigWrapper, DatabaseConfigWrapper, SchemaInstaller}
 import org.slf4j.LoggerFactory
 
-case class InstallSchema() extends Script {
+case class InstallSchema() extends SchemaInstaller with Script {
```

Comment:
It's just a matter of personal preference, but I think that Script with SchemaInstaller just reads "better", first of all it's a cli-script, and only after that it's schema installer. From scalac point of view it's almost identical (initialization order doesn't matter in this case)